packages/flutter_tools/lib/src/base/terminal.dart

```diff
@@ -90,16 +91,29 @@ class AnsiTerminal {
   /// Prompts the user to input a chraracter within the accepted list.
   /// Reprompts if inputted character is not in the list.
   ///
+  /// `prompt` is the text displayed prior to waiting for user input each time.
+  /// `defaultChoiceIndex`, if given, will be the character in `acceptedCharacters`
+  ///     in the index given if the user presses enter without any key input.
+  /// `displayAcceptedCharacters` prints also the accepted keys next to the `prompt` if true.
+  ///
   /// Throws a [TimeoutException] if a `timeout` is provided and its duration
   /// expired without user input. Duration resets per key press.
   Future<String> promptForCharInput(
-    List<String> acceptedCharacters, {
+    @required List<String> acceptedCharacters, {
     String prompt,
+    int defaultChoiceIndex,
     bool displayAcceptedCharacters: true,
     Duration timeout,
   }) async {
     assert(acceptedCharacters != null);
     assert(acceptedCharacters.isNotEmpty);
+    List<String> charactersToDisplay = acceptedCharacters;
+    if (defaultChoiceIndex != null) {
+      charactersToDisplay = new List<String>.from(charactersToDisplay);
+      charactersToDisplay[defaultChoiceIndex] = bolden(charactersToDisplay[defaultChoiceIndex]);
```

Comment:
this syntax annoys me. Wish there's a way to do charactersToDisplay[defaultChoiceIndex].=do(bolden) or something